docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,75 @@
+How To Set Orchestrator Session Limits
+======================================
+
+What Are Orchestrator Sessions?
+******************************
+
+By default an Orchestrator node can handle a maximum of **10** incoming livestreams at a time. Whenever the number of concurrent transcoding sessions go above this, the Orchestrator returns an ``OrchestratorCapped`` error to the Broadcaster node.
+
+Of course, the transcoding capacity and the bandwidth an Orchestrator has varies for everyone - so you can set the ``-maxSessions`` cli parameter with the correct number to fully utilize their hardware and get maximum work from the network.
+
+For example if an Orchestrator has enough resources (i.e. GPUs/CPUs/Bandwidth) to handle ``30`` streams instead you can pass it while starting up the node -
+
+::
+
+    $ livepeer -orchestrator -transcoder ... -maxSessions 30
+
+How To Calcuate My Capacity?
+****************************
+
+Calculate the session limit due to (1) transcoding hardware and (2) bandwidth as explained below, and take the minimum of the two. Finally, pass it through the ``-maxSessions`` parameter to your node as explained above.
+
+If you use a different Adaptive Bitrate (ABR) ladder configuration than the most common one referred in the steps, please follow the calculations similarly with your configuration.
+
+The bandwidth and computational power needed to transcode a video stream varies with the source video and output renditions' resolutions, bitrates and framerates. Thus any capacity estimate will only serve as a ballpark and you may want to tweak it after some real use on the network.
+
+**1. Transcoding Hardware Capacity**
+
+The ``livepeer_bench`` tool can help you get a rough idea of the number of concurrent transcoding sessions your hardware can handle. Refer to the :doc:`Benchmarking Guide <benchmark_transcoding>` on how to setup the tool and to learn more about the cli parameters. 
+
+Once you've got the tool setup - benchmark with the most common transcoding `ABR ladder <https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer_bench/transcodingOptions.json>`_ (``transcodingOptions.json``) with an initial guess for the concurrent sessions that you think your hardware can handle (suppose 15) -
```

Comment:
Do you think it would be easier to recommend a simple bash script to somewhat automate this?

Example pseudocode:

```
for i in range 0...20:
    ./livepeer_bench ... -concurrentSessions i | grep "Took" > bench.log
```

And then bench.log would contain the "Took..." statements for each run. And recommend changing 20 to a higher value if needed.

I also wonder how easy it would be incorporate this functionality into `livepeer_bench` itself...worth considering for the future.

Reply:
Yeah adding this within `livepeer_bench` sounds much better. For now I've added a small bash script in the docs.